src/Windows/GeneratePackage/InstallAzureDCAP.ps1

```diff
@@ -0,0 +1,31 @@
+param (
+    [Parameter(Mandatory=$true)][string]$localPath
+)
+
+if (-not (Test-Path -Path $localPath))
+{
+    Write-Host "[ERROR]: $localPath does not exist, please provide a valid path"
+}
+else
+{
+    Copy-Item "..\dll\dcap_quoteprov.dll" -Destination $localPath
+    Copy-Item "..\dll\libcurl.dll" -Destination $localPath
```

Comment:
Why are you copying all these DLLs? None of them should be needed.

Reply:
Removed all this.